Multisig/UI/UI Library/ContractVersionCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ContractVersionCell.swift
+//  Multisig
+//
+//  Created by Moaaz on 5/7/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct ContractVersionCell: View {
```

Comment:
now we have many duplications between contract version, safe cell, etc.

I'll ask you to rework it later.